doc/classes/World2D.xml

```diff
@@ -16,7 +16,7 @@
 			The [RID] of this world's canvas resource. Used by the [RenderingServer] for 2D drawing.
 		</member>
 		<member name="direct_space_state" type="PhysicsDirectSpaceState2D" setter="" getter="get_direct_space_state">
-			Direct access to the world's physics 2D space state. Used for querying current and potential collisions. Must only be accessed from the main thread within [code]_physics_process(delta)[/code].
+			Direct access to the world's physics 2D space state. Used for querying current and potential collisions. When using multi-threaded physics, access is limited to [code]_physics_process(delta)[/code] in the main thread.
```

Comment:
Should that new sentence be added to the `World3D` docs too?

Reply:
No, that's the point. There is no multi-threaded physics in 3D; so there's no need to restrict access to `PhysicsDirectSpaceState` to only the `_physics_process()` step.